com.adsbynimbus.nimbus/Editor/AndroidPostBuildProcessor.cs

```diff
@@ -100,6 +116,26 @@ public void OnPostGenerateGradleAndroidProject(string path) {
 				apsBuildWriter.Close();
 			#endif
 		}
+		private static void WriteGradleProps(string gradleFile) {
+			var propWriter = File.AppendText(gradleFile);
+			propWriter.WriteLine(@"
+				android.useAndroidX=true
+				");
+			propWriter.Flush();
+			propWriter.Close();
+		}
+
+		private static void RunEdm4uCheck(string path)
+		{
+			if(!File.ReadAllText(path + "/../gradle.properties").Contains("nimbus"))
```

Comment:
A better way to determine if EDM4U is used is to [List out all the installed packages](https://docs.unity3d.com/ScriptReference/PackageManager.Client.List.html) and search for the [PackageInfo.packageId](https://docs.unity3d.com/ScriptReference/PackageManager.PackageInfo.html) = `com.google.external-dependency-manager`